app/konnect/org-management/roles-reference.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Roles reference
+no_version: true
+---
+
+A team can have any number of roles. Roles can be assigned to teams, but not
+directly to users. See [Manage Teams, Roles, and Users](/konnect/org-management/teams-roles-users).
+
+The following predefined roles are available in Konnect:
+
+## Services
+
+| Role                        | Description  |
+|-----------------------------|--------------|
+| Admin | Admin of an existing Konnect Service. The admins have all write access permissions related to a Service and Service versions. |
+| Application Registration |  Allows user to enable or disable application registration for a Konnect Service. |
+| Creator | Create new Konnect Services in Service Hub. The creator becomes the owner of the Service they create, gaining admin access to the Service. This role is not for creating sub-entities in a Service such as Service versions, implimentations, API specs, or plugins. |
+| Deployer | Allows a user to impliment and associate a Konnect Service version to a runtime group. The user must also have permissions to deploy Services in the associated runtime group. |
+| Maintainer | Allows a user to read, edit, and deploy a Konnect Service and its Service versions, and manage its plugins. |
+| Plugins Admin | Allows a user to install plugins on the Konnect Service versions and Routes. The user must also have permissions to write to the associated runtime group. |
+| Publisher | Allows a user to publish a Konnect Service to the Dev Portal. The user must also have permissions in Portal to Publish Services in order to publish a service to the portal. |
+| Viewer | Read-only access to all the configurations of a Konnect Service, including attributes, versions, vitals reports, and plugins. |
+
+## Runtime groups
+
+| Role                        | Description  |
+|-----------------------------|--------------|
+| Admin | Owner of an existing runtime group. The owners have all write access related to a runtime group, the group's runtime instances, and its configuration. |
+| Creator | Creates a new runtime group in Runtime Manager. The creator becomes the owner of the runtime group they create, gaining admin access to the runtime group. This role does not directly grant permissions for creating runtime instances or configuring existing runtime groups. |
```

Comment:
`This role does not directly grant permissions for creating runtime instances or configuring existing runtime groups.` -> Which role do I need to create runtime instances? How about editing runtime groups?